spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-client-reactive-discovery-client-it/src/main/resources/application.yaml

```diff
@@ -1,3 +1,7 @@
+spring:
```

Comment:
so that `/actuator/heath` and `/services` are accessible on the same base. This is needed because traefik ingress does not seem to have a way to re-write the URL. As said before, this is not a big deal, since the test is supposed to test something else, not ingress rules